tpgtools/property.go

```diff
@@ -478,6 +478,41 @@ func createPropertiesFromSchema(schema *openapi.Schema, typeFetcher *TypeFetcher
 			p.customName = cn.Name
 		}
 
+		if p.Type.String() == SchemaTypeMap {
+			e := "&schema.Schema{Type: schema.TypeString}"
+			p.Elem = &e
+			p.ElemIsBasicType = true
+		}
+
+		if sens, ok := v.Extension["x-dcl-sensitive"].(bool); ok {
+			p.Sensitive = sens
+		}
+
+		if sendEmpty, ok := v.Extension["x-dcl-send-empty"].(bool); ok {
+			p.sendEmpty = sendEmpty
+		}
+
+		if v, ok := v.Extension["x-dcl-conflicts"].([]interface{}); ok {
+			for _, ci := range v {
+				p.ConflictsWith = append(p.ConflictsWith, ci.(string))
+			}
+		}
+
+		// Do this before handling properties so we can check if the parent is readOnly
+		isSGP := false
+		if sgp, ok := v.Extension["x-dcl-server-generated-parameter"].(bool); ok {
+			isSGP = sgp
+		}
+		if v.ReadOnly || isSGP || (parent != nil && parent.Computed) {
```

Comment:
I juggled around this code to be able to set `Computed` before handling sub-properties so that the sub-properties can check if their parent is `Computed` and inherit it

Reply:
Thanks!